src/io/flutter/jxbrowser/JxBrowserManager.java

```diff
@@ -39,12 +41,12 @@
 // the class path.
 public class JxBrowserManager {
   private static JxBrowserManager manager;
-  private static final String DOWNLOAD_PATH = PathManager.getPluginsPath() + File.separatorChar + "flutter-intellij" + File.separatorChar + "jxbrowser";
+  protected static final String DOWNLOAD_PATH = PathManager.getPluginsPath() + File.separatorChar + "flutter-intellij" + File.separatorChar + "jxbrowser";
   private static final AtomicReference<JxBrowserStatus> status = new AtomicReference<>(JxBrowserStatus.NOT_INSTALLED);
   private static final Logger LOG = Logger.getInstance(JxBrowserManager.class);
   // We will be gating JxBrowser features until all of the features are landed.
   // To test JxBrowser, set this to true and also add license key to VM options (-Djxbrowser.license.key=<key>).
-  public static final boolean ENABLE_JX_BROWSER = false;
+  public static final boolean ENABLE_JX_BROWSER = true;
```

Comment:
did you mean to leave this on as part of the CL?

Reply:
I thought I was adding these testing changes in a separate branch but accidentally committed them.